GeoPol.xml

```diff
@@ -17,7 +17,5 @@
     <ExcludeFolder>.gitignore</ExcludeFolder>
     <ExcludeFolder>GeoPol.xml</ExcludeFolder>
     <ExcludeFolder>THIRDPARTYNOTICES.md</ExcludeFolder>
-    <ExcludeFolder>src\Microsoft.Health.Fhir.Stu3.Core\Features\Definition\search-parameters.json</ExcludeFolder>
-    <ExcludeFolder>src\Microsoft.Health.Fhir.R4.Core\Features\Definition\search-parameters.json</ExcludeFolder>
```

Comment:
These files moved. Actually the rule is probably easier now, can you exclude all json files in https://github.com/microsoft/fhir-server/tree/master/src/Microsoft.Health.Fhir.Core/Data?

Reply:
Yes @brendankowitz - I'll update this pr with that exclusion.